.github/actions/cache-path/action.yml

```
@@ -0,0 +1,53 @@
+name: Cache path on host using github actions cache
+
+description: Add CHANNEL to GITHUB_ENV
+inputs:
+  cache-path:
+    description: "If Caching is used provide valid path"
+    default: ""
+    type: string
+  cache-key:
+    description: "If Caching is used provide valid key"
+    default: ""
+    type: string
+  cache-miss-script:
+    description: "Script to run if cache is missed"
+    default: ""
+    type: string
+  repository:
+    description: 'Repository to checkout, defaults to ""'
+    default: ""
+    type: string
+
+runs:
+  using: composite
+  steps:
+    # Use the same trick from https://github.com/marketplace/actions/setup-miniconda
+    # to refresh the cache daily. This is kind of optional though
+    - name: Get date
+      id: get-date
+      shell: bash
+      run: |
+        echo "today=$(/bin/date -u '+%Y%m%d')d" >> "${GITHUB_OUTPUT}"
+    - name: Run cache action
+      if: ${{ inputs.cache-path != '' }}
+      id: run-cache-action
+      uses: actions/cache@v3
+      with:
+        path: '${{ github.workspace }}//${{ inputs.repository }}${{ inputs.cache-path }}'
```

Why doubling forward slash?
```suggestion
        path: '${{ github.workspace }}/${{ inputs.repository }}${{ inputs.cache-path }}'
```

---

Done